assets/dev/scss/frontend/widgets/spacer.scss

```diff
@@ -5,14 +5,11 @@
 	}
 }
 
-.e-con {
-	--container-widget-width: 100%;
-}
-
```

Comment:
This affected other widgets, so I moved it down.

Reply:
Good call